### PR TITLE
Sync mainnet-eu config

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

The changes are what's in effect in mainnet-eu.  Except that this PR also disables acceptance tests in mainnet-eu citus namespace.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: clusters/mainnet-eu/mainnet-citus/helmrelease.yaml [hunk 1]
```diff
@@ -100,7 +112,7 @@ spec:
                 feature:
                   rejectToken: false
       cucumberTags: "@critical and not @nft"
-      enabled: true
+      enabled: false
```
Comment (xin-hedera) [inline clusters/mainnet-eu/mainnet-citus/helmrelease.yaml:115]: based on we have an operator account funding issue right now

**Review by steven-sheehy — DISMISSED** [review]

File: clusters/mainnet-eu/common/helmrelease.yaml [hunk 2]
```diff
@@ -60,6 +60,8 @@ spec:
           replicaExternalLabelName: __replica__
           resources:
             limits:
+              memory: 12Gi
+            requests:
               memory: 6Gi
```
Comment (jnels124) [inline clusters/mainnet-eu/common/helmrelease.yaml:65]: requests currently 10 but 6 is probably fine

File: clusters/mainnet-eu/mainnet/helmrelease.yaml [hunk 3]
```diff
@@ -38,6 +38,7 @@ spec:
     grpc:
       enabled: false
     importer:
+      enabled: false
```
Comment (jnels124) [inline clusters/mainnet-eu/mainnet/helmrelease.yaml:41]: we want this off?

**Review by jnels124 — DISMISSED** [review]

File: clusters/mainnet-eu/mainnet/helmrelease.yaml (hunk 3 above)
Comment (steven-sheehy) [reply clusters/mainnet-eu/mainnet/helmrelease.yaml:41]: Yes, hit the limit

Comment (jnels124) [reply clusters/mainnet-eu/mainnet/helmrelease.yaml:41]: oops mistread the namespace lol

File: clusters/mainnet-eu/common/helmrelease.yaml (hunk 2 above)
Comment (xin-hedera) [reply clusters/mainnet-eu/common/helmrelease.yaml:65]: will fix

**Review by jnels124 — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**